healthtools/config.py

```diff
@@ -26,6 +27,7 @@
 
 TEST_DIR = os.getcwd() + "/healthtools/tests"
 
-SLACK = {
-    "url": os.getenv("WEBHOOK_URL")
-    }
+BATCH = 5
+HF_BATCH = 100  # batch for health facilities
+
+DATA_DIR = os.getcwd() + "/healthtools/"
```

Comment:
Is `/data` or `/healthtools`? - https://github.com/CodeForAfrica-SCRAPERS/healthtools_ke/pull/13/files#diff-04c6e90faac2675aa89e2176d2eec7d8R30